airflow-core/src/airflow/ui/src/layouts/Nav/LogoutModal.tsx

```diff
@@ -31,8 +31,12 @@ const LogoutModal: React.FC<LogoutModalProps> = ({ isOpen, onClose }) => (
   <ConfirmationModal
     header="Logout"
     onConfirm={() => {
+      const baseHref = document.querySelector("head>base")?.getAttribute("href") ?? "";
+      const baseUrl = new URL(baseHref, globalThis.location.origin);
+      const logoutPath = new URL("api/v2/auth/logout", baseUrl).pathname;
+
       localStorage.removeItem(TOKEN_STORAGE_KEY);
-      globalThis.location.replace(`/api/v2/auth/logout`);
+      globalThis.location.replace(logoutPath);
```

Comment:
Nit: I think we should extract a small utility function for this piece of code. To not duplicate it from the `main.tsx` file.

Basically something like. `getRedictPath(targetPath)`. (That will read the tag and take this into account.

Reply:
Yeah sure we can do that. But you mean only for this one occurence right? Or reuse that function across main.tsx and this one?

Reply:
The utility function should be used across main and this one. We can merge that now to fix the issue, and we can always cleanup in follow up PRs if you'd like

Reply:
Yeah makes sense. Let me clean it in a follow uo